WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -746,7 +763,6 @@ nick_the_pale_death = {
 }
 nick_porphyrogenitos = {
 	allow = {
-		culture_group = byzantine
```

Comment:
The nickname goes to holders of the BYZANTINE or ROMAN empires that are BORN IN THE PURPLE. _ANY_ child of an Emperor can be Born In the Purple, and therefore, should be able to become famous for it! It's common sense.